docs/completion.md

```diff
@@ -23,7 +23,7 @@ On a Mac, install with `brew install bash-completion`
 
 Place the completion script in `/etc/bash_completion.d/` (`/usr/local/etc/bash_completion.d/` on a Mac), using e.g.
 
-     curl -L https://raw.githubusercontent.com/docker/compose/$(docker-compose --version | awk '{print $2}')/contrib/completion/bash/docker-compose > /etc/bash_completion.d/docker-compose
+     curl -L https://raw.githubusercontent.com/docker/compose/$(docker-compose --version | awk '{print $3}' | head -n1)/contrib/completion/bash/docker-compose > /etc/bash_completion.d/docker-compose
```

Comment:
The version capture should be changed to `docker-compose --version | awk 'NR==1{print $NF}'` which works for both v1.2.0 and v1.3.0.